docs_sources/erp_steuerung_durch_le-source.adoc

```diff
@@ -43,6 +45,7 @@ Der E-Rezept-Fachdienst speichert den Task unter einer generierten ID, welche im
 |===
 |URI        |https://erp.zentral.erp.splitdns.ti-dienste.de/Task/$create
 |Method     |POST
+|Berechtigt für |image:{PVS}[]
```

Comment:
URI, Method, HTTP Header sind englisch und "Berechtigt für" ist deutsch. Kann man das vereinheitlichen?

Reply:
In Target umbenennen